### PR TITLE
DDLS: Use Official Terms for "sourceType" Attribute

### PR DESCRIPTION
**Pull request by BeckerWdf**
The official term for source type "v" is "DDIC-Based Views". So let's also use it here.

### CODE REVIEW

**Comment by schneidermic0** [comment]
@BeckerWdf I understood you also wanted to rename the actual enum value "view", didn't you?

This would be an incompatible change, however since DDLS is not yet used, I think we can still change it.

File: file-formats/ddls/ddls-v1.json [hunk 1]
```diff
@@ -129,7 +129,7 @@
         "Projection View"
       ],
       "enumDescriptions": [
-        "View",
+        "DDIC-Based Views",
         "View Entity",
         "View Extend",
         "View Entity Extend",
```
Comment (schneidermic0) [inline file-formats/ddls/ddls-v1.json:135]: Not really  related to the actual PR. But I see descriptions are title style. They shall be sentence style. This is true for the whole DDLS file format.

```suggestion
        "DDIC-based views",
        "View entity",
        "View extend",
        "View entity extend",
```

@BeckerWdf Do you want to update all descriptions in this PR or shall I create an separate issue?

File: file-formats/ddls/type/zif_aff_ddls_v1.intf.abap [hunk 2]
```diff
@@ -15,9 +15,9 @@ INTERFACE zif_aff_ddls_v1
...
     "! Source type
     BEGIN OF co_source_type,
-      "! <p class="shorttext">View</p>
-      "! View
-      view               TYPE ty_source_type VALUE 'V',
+      "! <p class="shorttext">DDIC-Based Views</p>
+      "! DDIC-Based Views
+      ddic_based_view    TYPE ty_source_type VALUE 'V',
       "! <p class="shorttext">View Entity</p>
       "! View Entity
```
Comment (schneidermic0) [inline file-formats/ddls/type/zif_aff_ddls_v1.intf.abap:22]:
```suggestion
      "! DDIC-based Views
      ddic_based_view    TYPE ty_source_type VALUE 'V',
      "! <p class="shorttext">View Entity</p>
      "! View entity
```

**Review by schneidermic0 — CHANGES_REQUESTED** [review]
Thank you,  @BeckerWdf, for updating the file format. 👍 

In general the changes are fine, because we can still change incompatible for DDLS.

I just found a minor issue related to the descriptions in general. See comment below

File: file-formats/ddls/ddls-v1.json (hunk 1 above)
Comment (BeckerWdf) [reply file-formats/ddls/ddls-v1.json:135]: @schneidermic0: Let's handle this in a separate issue.

Comment (schneidermic0) [reply file-formats/ddls/ddls-v1.json:135]: I have just created a new issue: #441

File: file-formats/ddls/type/zif_aff_ddls_v1.intf.abap (hunk 2 above)
Comment (schneidermic0) [reply file-formats/ddls/type/zif_aff_ddls_v1.intf.abap:22]: See #441

**Review by schneidermic0 — APPROVED** [review]
We fix descriptions with #441. All other changes look fine to me.

**Pull request merged**